### DISCUSSION TITLE
Contact email

### DISCUSSION THREAD

**Discussion by marchoeppner** [bio-raum/FooDMe2#53] [General]
I am wondering if we need to have a contact Email in the MultiQC reports. If we do keep it, it should probably be a functional adress, not a personal one. Thoughts?

**Comment by gregdenay** [comment]
Do we actually need an email there? 
Wouldn't it be better if people adressed issues in the repo directly? 
Otherwise the best would probably be to to setup a foodme2@etwas adress

**Reply by marchoeppner** [reply]
I am leaning towards not having an Email in there - so I'll just take it out ;) If people complain, we can think again how we want to adress this.
